src/Backend/OnboardingInsuranceAPI/Areas/Insurance/GetInsurance.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using OnboardingInsuranceAPI.Services;
+
+namespace OnboardingInsuranceAPI.Areas.Insurance;
+
+public class GetInsurance : IHandler
+{
+    private readonly DataContext _context;
+    public GetInsurance(DataContext context)
+    {
+        _context = context;
+    }
+
+    public async Task<InsuranceData> GetInsuranceByPid(string pid)
+    {
+        var context =  await _context.Users.FirstOrDefaultAsync(u => u.Pid == pid);
```

Comment:
Name of this variable doesn't make sense.

Reply:
Oh, yeah, context should be users...

Reply:
it should be "user". Because you are calling first or default method.